builtin/logical/aws/secret_access_keys.go

```diff
@@ -226,7 +261,18 @@ func (b *backend) secretAccessKeysCreate(
 		return logical.ErrorResponse(err.Error()), nil
 	}
 
-	username, usernameWarning := genUsername(displayName, policyName, "iam_user")
+	defaultUserNameTemplate := `{{ printf "vault-%s-%s-%s-%s" (.DisplayName) (.PolicyName) (unix_time) (random 20) | truncate 64 }}`
```

Comment:
nit: the pattern other plugins have been using is to set this as a const at the top of the file.